OnboardIos/OnboardIos/Models/Users.m

```diff
@@ -31,74 +40,124 @@ + (NSArray *)list:(NSNumber *)page{
     [u6 setValue:@"rashimot" forKey:@"last_name"];
     [u6 setValue:@"2https://s3.amazonaws.com/uifaces/faces/twitter/calebogden/128.jpg" forKey:@"avatar"];
     [users addObject:u6];
     
     NSMutableDictionary *u7 = [NSMutableDictionary new];
-    [u7 setValue:@2 forKey:@"id"];
+    [u7 setValue:@7 forKey:@"id"];
     [u7 setValue:@"tab" forKey:@"first_name"];
     [u7 setValue:@"japa" forKey:@"last_name"];
     [u7 setValue:@"2https://s3.amazonaws.com/uifaces/faces/twitter/calebogden/128.jpg" forKey:@"avatar"];
     [users addObject:u7];
     
     NSMutableDictionary *u8 = [NSMutableDictionary new];
-    [u8 setValue:@2 forKey:@"id"];
+    [u8 setValue:@8 forKey:@"id"];
     [u8 setValue:@"Bianca" forKey:@"first_name"];
     [u8 setValue:@"alt" forKey:@"last_name"];
     [u8 setValue:@"2https://s3.amazonaws.com/uifaces/faces/twitter/calebogden/128.jpg" forKey:@"avatar"];
     [users addObject:u8];
     
     NSMutableDictionary *u9 = [NSMutableDictionary new];
-    [u9 setValue:@2 forKey:@"id"];
+    [u9 setValue:@9 forKey:@"id"];
     [u9 setValue:@"Felipe" forKey:@"first_name"];
     [u9 setValue:@"souza" forKey:@"last_name"];
     [u9 setValue:@"2https://s3.amazonaws.com/uifaces/faces/twitter/calebogden/128.jpg" forKey:@"avatar"];
     [users addObject:u9];
     
     NSMutableDictionary *u10 = [NSMutableDictionary new];
-    [u10 setValue:@2 forKey:@"id"];
+    [u10 setValue:@10 forKey:@"id"];
     [u10 setValue:@"joao" forKey:@"first_name"];
     [u10 setValue:@"das neves" forKey:@"last_name"];
     [u10 setValue:@"2https://s3.amazonaws.com/uifaces/faces/twitter/calebogden/128.jpg" forKey:@"avatar"];
     [users addObject:u10];
     
+    NSMutableDictionary *u11 = [NSMutableDictionary new];
+    [u11 setValue:@10 forKey:@"id"];
+    [u11 setValue:@"rafekl" forKey:@"first_name"];
+    [u11 setValue:@"matos" forKey:@"last_name"];
+    [u11 setValue:@"2https://s3.amazonaws.com/uifaces/faces/twitter/calebogden/128.jpg" forKey:@"avatar"];
+    [users addObject:u11];
+    NSMutableDictionary *u12 = [NSMutableDictionary new];
+    [u12 setValue:@10 forKey:@"id"];
+    [u12 setValue:@"guiilernmes" forKey:@"first_name"];
+    [u12 setValue:@"afonso" forKey:@"last_name"];
+    [u12 setValue:@"2https://s3.amazonaws.com/uifaces/faces/twitter/calebogden/128.jpg" forKey:@"avatar"];
+    [users addObject:u12];
+    NSMutableDictionary *u13 = [NSMutableDictionary new];
+    [u13 setValue:@10 forKey:@"id"];
+    [u13 setValue:@"mikey" forKey:@"first_name"];
+    [u13 setValue:@"pateat" forKey:@"last_name"];
+    [u13 setValue:@"2https://s3.amazonaws.com/uifaces/faces/twitter/calebogden/128.jpg" forKey:@"avatar"];
+    [users addObject:u13];
     return users;
 }
 
 - (void)incrementViewCount:(NSNumber *)identifier {
     DLog();
+    
+    BOOL create = YES;
+    for (int a = 0; a < userViewCount.count; a++) {
+        NSDictionary *dict = [userViewCount objectAtIndex:a];
+        if (dict[@"id"]==identifier) {
```

Comment:
As you are using this keys for the dictionary, lots of times, if anyone ask you to change any key, you will have a trouble, so because of this I strongly recommend you extract this, to a constants file, or put at the begin of your class, all declarations, and just use it

this keep the maintenance easier